Stanford/JumpstartEngineering/Install/Users/WMDUsersRoles.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * @file
+ * Create and update users; map roles
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Users;
+/**
+ *
+ */
+class WMDUsersRoles extends \AbstractInstallTask {
+
+  /**
+   * Create and update users; map roles
+   *
+   * @param array $args
+   *   Installation arguments.
+   *
+   * @todo: Refactor this into several methods and/or a generic Utility class.
+   */
+  public function execute(&$args = array()) {
+
+    // Need this for UI install.
+    require_once DRUPAL_ROOT . '/includes/password.inc';
+    $install_vars = variable_get('stanford_jumpstart_install', array());
+    $config_form_data = $args['forms']['install_configure_form'];
+    // Get some stored variables.
+    if ($args['interactive']) {
```

Comment:
This whole block needs to be cleaned up to just use the install_vars. No drush_options any more.